.github/workflows/cpp-ci.yml

```diff
@@ -50,6 +54,14 @@ jobs:
         run: cmake --build build --config Release
       - name: Test
         run: ctest -C Release --output-on-failure --test-dir build --repeat until-pass:3 --timeout 400
+      - name: Export variables for next jobs
+        uses: UnlyEd/github-action-store-variable@v3 # See https://github.com/UnlyEd/github-action-store-variable
+        with:
+          # Persist (store) our MAGIC_NUMBER ENV variable into our store, for the next jobs
+          variables: |
+            gurobiVersion=${{ env.gurobiVersion }}
+            gurobiShortVersion=${{ env.gurobiShortVersion }}
+            gurobiFolder=${{ env.gurobiFolder }}
```

Comment:
Instead of storing the variables here and retrieving them in the other jobs, couldn't you simply use the other action again that reads the JSON file? That would simplify/unify the whole configuration a little.
Just a suggestion though. This seems to work just fine.

Reply:
@burgholzer Unfortunately the action to read json only works on ubuntu, that's why I had to use this workaround... :/